controller/eks-cluster-config-handler.go

```diff
@@ -248,6 +248,14 @@ func (h *Handler) OnEksConfigRemoved(_ string, config *eksv1.EKSClusterConfig) (
 		}
 	}
 
+	if aws.BoolValue(config.Spec.EBSCSIDriver) == true {
+		logrus.Infof("deleting ebs csi driver role for config [%s]", config.Name)
+		err = deleteStack(h.awsServices.cloudformation, getEBSCSIDriverRoleStackName(config.Spec.DisplayName), getEBSCSIDriverRoleStackName(config.Spec.DisplayName))
```

Comment:
```suggestion
		if err := deleteStack(h.awsServices.cloudformation, getEBSCSIDriverRoleStackName(config.Spec.DisplayName), getEBSCSIDriverRoleStackName(config.Spec.DisplayName)); err != {
```
I know inline ifs are not consistently used in the operator code but it's a golang idiomatic way and we can start doing it